### DISCUSSION TITLE
Публикация на инфостарте

### DISCUSSION THREAD

**Discussion by tahallus** [cpr1c/it#2] [General]
Почему не публикуешь на инфостарте?
Продукт хороший, больше узнают людей о нем, будут помогать с развитием.
Как пример BlizD.

**Comment by LevkinSergey** [comment]
никак руки не дойдут статью написать. До этого у себя переходили. Нужно было понять насколько это юзабельно. 

Планирую в мае выложиться.

**Reply by tahallus** [reply]
И как вышло юзабельно ? или решили отказаться от системы и используете что-то другое ?

**Reply by LevkinSergey** [reply]
Сами пользуемся. До этого был редмайн. То что в нем использовали, есть в этой системе. И плюсом можем сами допиливать.
Есть мысли по улучшению, но пока не до нее.
